notebook/files/handlers.py

```diff
@@ -12,12 +12,19 @@
     from base64 import decodestring as decodebytes
 
 
-from tornado import web, escape
+from tornado import web
 
 from notebook.base.handlers import IPythonHandler
 
+
 class FilesHandler(IPythonHandler):
-    """serve files via ContentsManager"""
+    """serve files via ContentsManager
+
+    Normally used when ContentsManager is not a FileContentsManager.
+
+    FileContentsManager subclasses use AuthenticatedFilesManager by default,
+    a subclass of StaticFileManager.
```

Comment:
"AuthenticatedFilesManager" and "StaticFileManager" should be "...Handler", I think?

Reply:
I got Managers on the mind. Fixed.